common/test/unit/test_models.py

```diff
@@ -27,3 +33,124 @@ def test_metadata(self):
 
         self.assertEqual(metadata.get('parent_id'), 'xyz')
         self.assertEqual(metadata.get('size'), 1024)
+
+
+class TestDockerManifest(unittest.TestCase):
+    """
+    This class contains tests for the DockerManifest class.
+    """
+    def test___init__(self):
+        """
+        Assert correct operation of the __init__() method.
+        """
+        name = 'name'
+        tag = 'tag'
+        architecture = 'x86_65'  # it's one better
+        digest = 'sha256:6c3c624b58dbbcd3c0dd82b4c53f04194d1247c6eebdaab7c610cf7d66709b3b'
+        fs_layers = [{'layer_1': 'rsum:jsf'}]
+        history = [{'v1Compatibility': 'not sure what goes here but something does'}]
+        schema_version = 1
+        signatures = [{'some': 'signature'}]
+
+        m = models.DockerManifest(name, tag, architecture, digest, fs_layers, history,
+                                  schema_version, signatures)
+
+        self.assertEqual(m.name, name)
+        self.assertEqual(m.tag, tag)
+        self.assertEqual(m.architecture, architecture)
+        self.assertEqual(m.digest, digest)
+        self.assertEqual(m.fs_layers, fs_layers)
+        self.assertEqual(m.history, history)
+        self.assertEqual(m.signatures, signatures)
+        self.assertEqual(m.schema_version, schema_version)
+
+    def test___init___bad_schema(self):
+        """
+        Assert correct operation of the __init__() method with an invalid (i.e., != 1) schema
+        version.
+        """
+        name = 'name'
+        tag = 'tag'
+        architecture = 'x86_65'  # it's one better
+        digest = 'sha256:6c3c624b58dbbcd3c0dd82b4c53f04194d1247c6eebdaab7c610cf7d66709b3b'
+        fs_layers = [{'layer_1': 'rsum:jsf'}]
+        history = [{'v1Compatibility': 'not sure what goes here but something does'}]
+        schema_version = math.pi
```

Comment:
I was wondering why you would need to import ``math``. Now I see that you didn't, and all is clear. ;)